### PR TITLE
[WIP] Topic workspace overview

### PR DESCRIPTION
**Pull request by petra-sargent**
### What does this PR do?
This PR fixes style and grammar errors. This PR contains the what-are-workspaces.adoc file that was part of PR #508. The old PR #508 contains 3 files and will be closed. It is easier to have only one file per PR.

### What issues does this PR fix or reference?
#507

### CODE REVIEW

**Comment by petra-sargent** [comment]
@rkratky @robin-owen @preetic  Please review. Thank you.

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 1]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
 [id="workspace"]
 == Workspace
 
-Eclipse Che reinvents the definition of a workspace which is usually termed as a local directory with projects and some meta-information that IDE uses to properly configure them. In Eclipse Che, a workspace is the developer environment itself represented by Docker containers, k8s pods or something else (for example, a VM, localhost etc), coupled with data persistence mechanism, provision of environment variables, projects and commands which are associated with them, as well as resources allocation attributes.
+A workspace is usually termed as a local directory with projects and meta-information that the integrated development environment (IDE) uses to configure projects. In Eclipse Che, a workspace is the developer environment. The developer environment contains Docker containers, Kubernetes pods, and a virtual machine or local host.  Environment variables and storage volumes are part of the workspace. The developer environment also contains projects, project commands, and resource allocation attributes.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: local host -> localhost

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 2]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
 
 [id="environment"]
 == Environment
 
-Workspace runtime environment is a set of machines where each machine is defined by a recipe. Environment is considered to be healthy when all machines have successfully started and associated installers executed their jobs. Environment is defined by a recipe that can have different types. It is up to environment (and underlying infra) to validate a recipe.
+The workspace runtime environment is a set of machines where each machine is defined by a recipe. The environment is healthy when all the machines successfully start up and the installers execute jobs. The environment is defined by a recipe that can have different types.  The environment and infrastructure validates a recipe.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: * start up -> start (don't use phrasal verbs)
* The environment and infrastructure validates a recipe. -> The environment and underlying infrastructure validate a recipe.

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 3]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
 
 [id="machine"]
 == Machine
 
-Runtime environment has minimum one machine that can be (at this moment) a Docker container or a Kubernetes pod. You can create multi machine environments with as many machines as your project infrastructure requires. Though each machine has its own configuration and start policy, it is an integral part of environment, and start failures/crashes of one machine may signal about an entire environment being unhealthy. Machines can freely communicate in a common environment using internal network (e.g. by `service:port`).
+The runtime environment has a minimum of one machine that can be a Docker container or a Kubernetes pod. You can create multi-machine environments with as many machines as your project infrastructure requires. Each machine has a configuration and start policy.  Machine crashes and start failures are signs of an unhealthy environment.  Machines communicate by using the internal network, `service:port`.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: Docker container -> Docker-formatted container

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 4]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
 
 [id="recipe"]
 == Recipe
 
-Workspace environment is defined by a recipe that can be a single Docker image, a Dockerfile, a Docker Compose file, or a Kubernetes list of objects with multiple pods and services.
+A workspace environment is defined by a recipe.  The recipe can be one of the following:
+
+* single Docker image
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: Docker image -> container image

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 5]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
+* Docker file
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: -> Dockerfile (see https://docs.docker.com/get-started/part2/#define-a-container-with-dockerfile)

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 6]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
+* Docker Compose file
+* Kubernetes list of objects with multiple pods and services
 
 [id="bootstrapper"]
 == Bootstrapper
 
-Inside every machine, a bootstrapper is started. Its role is to start installer scripts that will install chosen software/components into a machine. Bootstrapper is a tiny binary compiled from go code. It starts with a set of parameters and a config file with the list of installer scripts to be executed (see below). Launch of a bootstrapper is the first process executed in a machine after `CMD` or `ENTRYPOINT`.
+The bootstrapper starts the installer script after the first process is executed in the machine following the `CMD` or `ENTRYPOINT`.   The role of the bootstrapper is to start the installer scripts with a set of parameters and a configuration file. The bootstrapper is a tiny binary compiled from Go code.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: * the `CMD` or `ENTRYPOINT` -> the `CMD` or `ENTRYPOINT` instructions
* tiny -> small

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 7]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
 
 [id="environment-variables"]
-== Environment Variables
+== Environment variables
 
-An individual set of environment variables is propagated into each individual machine. Depending on the infrastructure, envs are propagated to Docker containers or k8s pods.
+The environment variables are propagated into each individual machine. Depending on the infrastructure, environment variables are propagated to Docker containers or Kubernetes pods.
 
 [id="whats-next"]
 == What’s Next?
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: no contractions

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 8]
```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
...
 
-link:creating-starting-workspaces.html[Create and start your first workspace], learn how to define link:volumes.html[volumes] and link:env-variables.html[env variables].
-
+link:creating-starting-workspaces.html[Create and start your first workspace], learn how to define link:volumes.html[volumes] and link:env-variables.html[environment variables].
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: comma splice

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/workspace-admin/what-are-workspaces.adoc [hunk 9]
```diff
@@ -52,7 +52,8 @@ A volume is a fixed amount of storage that is used to persist workspace data.  W
 
 The environment variables are propagated into each individual machine. Depending on the infrastructure, environment variables are propagated to Docker containers or Kubernetes pods.
 
-[id="whats-next"]
-== What’s Next?
+[id="what-is-next"]
+== What is Next?
```
Comment (rkratky) [inline src/main/pages/workspace-admin/what-are-workspaces.adoc]: sentence case, please :)

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by rkratky** [comment]
Thanks @petra-sargent.

@eivantsov, @slemeur, please merge when convenient.

**Review by slemeur — APPROVED** [review]

**Pull request merged**